01_temperature/temperature.rb

```diff
@@ -0,0 +1,20 @@
+def ftoc(val)
+  val = val-32
+  5.0*val/9.0
+
+=begin
+  if val==32
+    0
+  else
+    5.0*(val-32)/9.0
+  end
+=end
+end
+
+def ctof(val)
+  if val == 0
```

Comment:
How can we rewrite this as a one liner?

Reply:
5.0*(val-32)/9.0

Reply:
Else statement not needed, when _val_ is 0 the result will be 9.0*0/5.0+32 = 32.
So, the return of this method should be  9.0*val/5.0 + 32

Reply:
We can also use `.zero?` method instead. https://ruby-doc.org/core-3.0.0/Integer.html#method-i-zero-3F.

Nice job noticing that the `if` isn't needed.

Tip: In Ruby, `if` statements can also be rewritten using the ternary operator. https://www.rubyguides.com/2019/10/ruby-ternary-operator/